docs/content/integrations/airbyte.mdx

```diff
@@ -0,0 +1,136 @@
+---
+title: "Using Dagster with Airbyte"
+description: Integrate your Airbyte connections into Dagster.
+---
+
+# Using Airbyte with Dagster
+
+Dagster can orchestrate your Airbyte connections, making it easy to chain an Airbyte sync with upstream or downstream steps in your workflow.
+
+This guide focuses on how to work with Airbyte connections using Dagster's [software-defined asset (SDA)](/concepts/assets/software-defined-assets) framework.
+
+---
+
+## Airbyte connections and Dagster software-defined assets
+
+An [Airbyte connection](https://docs.airbyte.com/understanding-airbyte/connections/) defines a series of data streams which are synced between a source and a destination. During a sync, each stream is replicated in the destination, typically as one or more tables. Dagster represents each of the replicas generated in the destination as a software-defined asset. This enables you to easily:
```

Comment:
"Dagster represents each replica" is the first time the word "replica" is used. it would be helpful to use it in a pure-Airbyte context first to make the mapping more clear.